frontend/amundsen_application/static/js/pages/TableDetailPage/ReportTableIssue/styles.scss

```diff
@@ -26,5 +26,14 @@
   .data-owner-notification {
     font-size: 10px;
     margin: 5px 0 0;
+    float: left;
+  }
+
+  .priority-btn-group {
+    float: left;
+  }
+
+  .issue-submit-btn {
+    float: right;
```

Comment:
This will probably will work fine, but we are tending to use flexbox nowadays instead of floats for positioning.